plugins/alignments/src/BamAdapter/MismatchParser.ts

```diff
@@ -150,18 +150,19 @@ export function mdToMismatches(
   }
 
   // now actually parse the MD string
-  const md = mdstring.match(/(\d+|\^[a-z]+|[a-z])/gi) || []
+  const md = mdstring.match(/(\d+|\^[A-Za-z]+|[A-Za-z])/g) || []
```

Comment:
is this a bugfix? you sure about the case insensitivity?

Reply:
went ahead and reverted. it was still case insensitive (see both A-Z and a-z) and thought it might be a bit faster based on benchmarking but it's approx equal